### PR TITLE
feat: publish record buttons

### PR DESCRIPTION
**Pull request by luke-mcfarlane-rocketlab**
## JIRA Ticket

[BSS-389](https://jira.csiro.au/browse/BSS-389)

## Description

Small update to styling, visibility and text of publish record buttons.

## Proposed Changes

- Updated styling, visibility and text of publish record buttons
- Fixed styling issue with form's parent component by removing it

## How to Test

1. Navigate to a survey
2. Add new (or open current) record
3. Scroll down to the buttons at the end of the form

## Checklist

- [x] I have confirmed all commits have been signed.
- [x] I have added JSDoc style comments to any new functions or classes.
- [x] Relevant documentation such as READMEs, guides, and class comments are updated.

### CODE REVIEW

**Review by stevecassidy — CHANGES_REQUESTED** [review]
The change seems to totally mess up layout on narrower screen widths.  Tried at 1000px wide and the form section shrinks to half the page width. At 590px has switched to narrow view but form is still 2/3 of screen width.

**Comment by luke-mcfarlane-rocketlab** [comment]
I wasn't able to replicate the issue mentioned above

**Comment by PeterBaker0** [comment]
I'll see if I can replicate

**Comment by PeterBaker0** [comment]
I can replicate the layout issue e.g. 
image

@luke-mcfarlane-rocketlab  to address this and can review after

**Comment by luke-mcfarlane-rocketlab** [comment]
Ready for re-review @PeterBaker0

File: app/android/app/src/main/AndroidManifest.xml-e
Comment (PeterBaker0) [inline app/android/app/src/main/AndroidManifest.xml-e:1]: should this be git ignored?

File: app/src/gui/components/record/formButton.tsx [hunk 1]
```diff
@@ -75,73 +78,52 @@ export default function FormButtonGroup(props: any) {
...
+          text="Publish and Close Record"
+          is_close={'close'}
+          handleFormSubmit={handleFormSubmit}
+          is_final_view={is_final_view}
+        />
+        <FormSubmitButton
+          color="secondary"
+          disabled={disabled}
+          formProps={formProps}
+          text="Publish and New Record"
```
Comment (PeterBaker0) [inline app/src/gui/components/record/formButton.tsx:121]: Was this supposed to be informed by the name of the viewset/form e.g. "Site" ?

**Review by PeterBaker0 — COMMENTED** [review]
Trusting that you've tested functionality - more important to get things merged at this point. Code looks like mostly styling/minor changes. Am concerned about hardcoding value of Publish button - should it say "Building" for example instead of "record"?

**Comment by luke-mcfarlane-rocketlab** [comment]
@PeterBaker0 actioned above

File: app/android/app/src/main/AndroidManifest.xml-e
Comment (PeterBaker0) [reply app/android/app/src/main/AndroidManifest.xml-e:1]: @luke-mcfarlane-rocketlab  can you remove this ? It has hardcoded build artifact outputs such as org.fedarch.faims3 etc.

File: app/src/gui/components/record/formButton.tsx (hunk 1 above)
Comment (PeterBaker0) [reply app/src/gui/components/record/formButton.tsx]: @luke-mcfarlane-rocketlab  did you action this?

File: app/android/app/src/main/AndroidManifest.xml-e
Comment (PeterBaker0) [inline app/android/app/src/main/AndroidManifest.xml-e:1]: can you git remove this sucker?

**Review by PeterBaker0 — APPROVED** [review]
Nit pick - but would be better to a) make viewName.split('-').at(0) into a function since it's a bit obtuse why that works b) store it as a constant in component scope rather than derive it twice directly in the text. 

Otherwise once the manifest file is removed lgtm 👍

**Pull request merged**